cmd/create/operatorroles/cmd.go

```diff
@@ -149,12 +149,21 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	if cluster.State() != cmv1.ClusterStatePending {
-		reporter.Errorf("Cluster '%s' is %s and does not need additional configuration.",
-			clusterKey, cluster.State())
+	// Check to see if IAM operator roles have already created
+	missingRoles, err := validateOperatorRoles(awsClient, cluster)
+	if err != nil {
+		reporter.Errorf("Unable to find validate operator roles exist: %s", err)
 		os.Exit(1)
 	}
 
+	if len(missingRoles) == 0 {
+		if cluster.State() != cmv1.ClusterStatePending {
+			reporter.Errorf("Cluster '%s' is %s and does not need additional configuration.",
+				clusterKey, cluster.State())
+			os.Exit(1)
```

Comment:
I wrote this originally, but now I'm wondering if it should be an `Infof` with `exit(0)`. What do you think?

Reply:
I think that makes sense, i'll update it